core/arch/arm/plat-k3/main.c

```diff
@@ -105,3 +106,4 @@ TEE_Result tee_otp_get_hw_unique_key(struct tee_hw_unique_key *hwkey)
 
 	return TEE_SUCCESS;
 }
+#endif
```

Comment:
Nit: Can we pleaser end the #endif with ``//PLATFORM_FLAVOR_am62lx`` ?
Also, while at it please update all files copyright dates.

Thanks,
Reviewed-by: Dhruva Gole <d-gole@ti.com>

Reply:
Nit in the nit ;-) prefer C-style comments `/* PLATFORM_FLAVOR_am62lx */`